examples/nginx.conf

```diff
@@ -31,23 +31,6 @@ http {
   # certs and keys. 1MB should be enough to store 100 certs/keys.
   js_shared_dict_zone zone=acme:1m;
 
-  ##
-  # Set up a specific server to listen on the localhost interface for
-  # js_periodic to trigger acme.clientAutoMode. The default URL is
-  # http://localhost:10999/acme/auto but can be overriden via the
-  # NJS_ACME_CLIENT_AUTO_MODE_URL environment variable.
-  server {
```

Comment:
A++ deletion here, this was all accidental complexity that we no longer have to explain to the user!